compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/methodgen/MethodGen.java

```diff
@@ -927,6 +944,9 @@ private void createLocalVariableTable(BIRFunction func, BIRVarToJVMIndexMap inde
             mv.visitLocalVariable("self", String.format("L%s;", B_OBJECT), null, methodStartLabel, methodEndLabel, 0);
         }
         BIRBasicBlock endBB = func.basicBlocks.get(func.basicBlocks.size() - 1);
+        if (endBB.terminator.thenBB != null) {
+            endBB = endBB.terminator.thenBB;
+        }
```

Comment:
just for my understanding, why is this change required?

Reply:
Sorry, forgot to remove before send the PR. Removed